src/Commands/CreateRLSPoliciesForTenantTables.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * Creates and uses RLS policies for tables of models related to a tenant directly (using BelongsToTenant),
+ * or through a parent primary model (using BelongsToPrimaryModel).
+ *
+ * The models are discovered in the directories configured in the Tenancy config ('tenancy.rls.model_directories').
+ *
+ * This command is used with Postgres + single-database tenancy.
+ */
+class CreateRLSPoliciesForTenantTables extends Command
+{
+    protected $signature = 'tenants:create-rls-policies';
+
+    public function handle(): int
+    {
+        DB::transaction(function () {
+            foreach (tenancy()->getTenantModels() as $model) {
+                $this->useRlsOnModel($model);
+            }
+        });
+
+        return Command::SUCCESS;
+    }
+
+    /**
+     * Make model use RLS if it belongs to a tenant directly, or through a parent primary model.
+     */
+    protected function useRlsOnModel(Model $model): void
```

Comment:
Maybe this method could have a more accurate name, something like "apply" or some other word that implies the persistent change we're making to the DB. "Use" sounds like it could be something temporary/something within the lifecycle of a single request.

Also, a small nitpick: we should probably use uppercase `RLS` consistently across class names and methods. Unless you think there's a good reason to use different casing for class names and methods.

Reply:
Yeah I thought about the casing too – "Rls" seemed more appropriate, but we can make it uppercase everywhere,

Reply:
Maybe keep this review on hold for now and I'll revisit it when the other things in the code are resolved.

Reply:
Used "apply" instead of "use" in the method name and changed all "Rls" instances to "RLS"